frontend/src/components/Topics/Topic/Messages/Filters/Filters.tsx

```diff
@@ -187,7 +284,84 @@ const Filters: React.FC<FiltersProps> = ({
           </Button>
         </FlexBox>
 
-        <Search placeholder="Search" value={search} onChange={setSearch} />
+        <FlexBox gap="8px" alignItems="center">
+          <Search placeholder="Search" value={search} onChange={setSearch} />
+          <div style={{ position: 'relative' }}>
+            <Button
+              disabled={isFetching || messages.length === 0}
+              buttonType="secondary"
+              buttonSize="M"
+              onClick={handleDownloadClick}
+              style={{
+                minWidth: '40px',
+                padding: '8px',
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+              }}
+            >
+              <svg
+                width="24"
```

Comment:
The icon is 16x16 in figma
<img width="402" height="187" alt="image" src="https://github.com/user-attachments/assets/f1eb5fac-843c-4490-8609-fb946f56e0c0" />

Reply:
<img width="397" height="185" alt="image" src="https://github.com/user-attachments/assets/5f5d1a88-47df-4c60-a808-b8d2b5be563c" />

Could you adjust the size please?

Reply:
@alexanderlz Any chance you can resume this work? 

I am looking forward to using this feature

Thanks again! 🎖️

Reply:
yes, will do it during one of the upcoming days

Reply:
Awesome, thank you for your hard work @alexanderlz 

@Haarolean Any chance you can take a look again? 

Thanks in advance 🫡